cmd/controlplane/controller.go

```diff
@@ -169,7 +169,7 @@ func newControllerCommand() *cobra.Command {
 			}
 
 			var rolloutsMgr manager.Manager
-			{
+			if types.MustParseBool(os.GetEnv("ROLLOUTS_INTEGRATION_ENABLED", "true")) {
```

Comment:
This is a spot where we might try to make an inference from the presence or absence of Rollouts CRDs.